Dockerfile

```diff
@@ -0,0 +1,21 @@
+ARG PHP_VERSION=7.2
+FROM php:${PHP_VERSION}-alpine
+
+RUN apk add --no-cache \
+	autoconf \
+	make \
+	g++ \
+	bash \
+	git \
+	openssl-dev
+
+RUN echo "memory_limit=2G" >> /usr/local/etc/php/php.ini
+RUN if [ ${PHP_VERSION:0:3} == "5.6" ] ; then pecl install mongo && echo "extension=mongo.so" >> /usr/local/etc/php/php.ini ; else pecl install mongodb && docker-php-ext-enable mongodb ; fi
```

Comment:
Would be better to make it explicit with another build argument? What do you think?

Reply:
Because we use TRAVIS_PHP_VERSION a build argument is not the correct solution here.  Comparing the substr of the PHP_VERSION works nicely.

Reply:
The point that @thomasvargiu is making is that we can provide this as a flag instead of a version number via a build argument in `docker-compose.yml` (as he demonstrates in [another comment](https://github.com/zfcampus/zf-apigility-doctrine/pull/308/files#r183233385)). If the initial `docker-compose (build|up --build)` operation includes appropriate ENV variables, these can be referenced via `docker-compose.yml` build arguments and referenced in the `Dockerfile`. That would  arguably be better than parsing versions in the `Dockerfile`, and make it more explicit the _scenarios_ we are testing.